services/brig/src/Brig/Team/API.hs

```diff
@@ -6,11 +6,12 @@ import Brig.API.User (fetchUserIdentity)
 import qualified Brig.API.User as API
 import Brig.App (currentTime, settings)
 import qualified Brig.Data.Blacklist as Blacklist
-import Brig.Data.UserKey (userEmailKey)
+import Brig.Data.UserKey
 import qualified Brig.Data.UserKey as Data
 import Brig.Email
 import qualified Brig.IO.Intra as Intra
 import Brig.Options (setMaxTeamSize, setTeamInvitationTimeout)
+import Brig.Phone
```

Comment:
Please import this qualified

Reply:
👍

Reply:
https://github.com/wireapp/wire-server/pull/1032/commits/f9d3a5f0840566c48f91d4f3fc81cd34aaf4e8d3